### DISCUSSION TITLE
Can I activate 2fa permanently?

### DISCUSSION THREAD

**Discussion by Trust996** [dani-garcia/vaultwarden#4325] [Q&A]
Hi everyone,
just a quick question:
I have tested 2fa via google authenticator and yubikey and it works perfectly. Unfortunately the browser only asks for 2fa once. After the device is registered, it is enough to enter the password. Can I configure the browser extension to ask for 2fa every time?
Thanks

**Comment by stefan0xC** [comment]
You can switch the [vault timeout action](https://bitwarden.com/help/vault-timeout/#vault-timeout-action) from lock to log out.

**Comment by stefan0xC** [comment]
You could also try enabling this option but I'm not sure how well it works with the browser extension:
https://github.com/dani-garcia/vaultwarden/blob/bb2412d0339e1da5dee99fc566a2b2aab5d2808c/.env.template#L453-L456

**Comment by Trust996** [comment]
Thank you very much, I will test both solutions.

**Reply by vidnick** [reply]
@Trust996 were you ever able to get this to request 2fa upon every login to the extension?

Follow up to that, were you also able to stop the email New Device Logged In From Chrome from sending each day?
